necst/core/commander.py

```diff
@@ -118,7 +153,7 @@ def wait_convergence(
                 error_az = self.parameters[ENC].lon - self.parameters[CMD].lon
                 error_el = self.parameters[ENC].lat - self.parameters[CMD].lat
                 if checker.check(
-                    error_az**2 + error_el**2 < threshold[target] ** 2
+                    error_az ** 2 + error_el ** 2 < threshold[target] ** 2
```

Comment:
```suggestion
                    error_az**2 + error_el**2 < threshold[target] ** 2
```

[Black](https://black.readthedocs.io/en/stable/) というコード整形ソフトを使っているため、変数名と冪乗演算子の間のスペースはエラーになります (右辺は変数名 `threshold` と演算子の間に辞書のキーワード指定 `[target]` が入っているため例外です)。
もしより良いフォーマット (整形ソフト) を見つけたら別のissueで提案してください。

Reply:
https://github.com/necst-telescope/internal/discussions/2
にて記載されている通りの VSCode 設定を行っていますが、なぜか空白ありにフォーマットされてしまっているようです（空白を消して Save しても、Save したときに勝手に空白が追加されてしまいます）。

Commit suggestion させていただきます。